prune.go

```diff
@@ -23,11 +23,19 @@ var DefaultFiles = []string{
 	".gitattributes",
 	".editorconfig",
 	".eslintrc",
+	".eslintrc.js"
+	".eslintignore",
+	".npmignore"
 	".jshintrc",
 	".flowconfig",
 	".documentup.json",
 	".yarn-metadata.json",
 	".travis.yml",
+	"circle.yml",
+	".coveralls.yml"
+	"CHANGES",
+	"CHANGELOG.md",
+	"HISTORY.md",
```

Comment:
The `*.md` already exist in `DefaultExtensions` list.

Reply:
my bad. somehow overlooked that! Fixed now